packages/jest-fake-timers/src/jestFakeTimers.ts

```diff
@@ -237,6 +237,20 @@ export default class FakeTimers<TimerRef> {
       .forEach(([timerHandle]) => this._runTimerHandle(timerHandle));
   }
 
+  advanceTimersToNextTimer(steps: number = 1): void {
```

Comment:
```suggestion
  advanceTimersToNextTimer(step = 1) {
```

it should infer the type correctly